### ISSUE TITLE
err

### ISSUE THREAD

**Issue by gavrushoves** [agragregra/gulp-lesson#9]
склонировал себе реп, в нем написал npm init и npm i, потом npm watch а он мне 
![image](https://user-images.githubusercontent.com/30019200/46865671-00f67b80-ce27-11e8-82fd-3fc4ec7aedcd.png)

**Comment by gavrushoves** [comment]
в чем может быть проблема?

**Comment by serhii-syritchenko** [comment]
> в чем может быть проблема?

в package.json есть модуль 'imagemin-pngquant'?

**Comment by gavrushoves** [comment]
да! я ничего не менял, как в авторском репозитории, так и у меня
![image](https://user-images.githubusercontent.com/30019200/46866273-60ee2180-ce29-11e8-81ee-00d0c2e5f332.png)

**Comment by ghost** [comment]
Gulpfile выложи

**Comment by ghost** [comment]
Либо заново установи imagemin-pngquant'

**Comment by gavrushoves** [comment]
![image](https://user-images.githubusercontent.com/30019200/46866634-dc040780-ce2a-11e8-9ffd-7797b817cd8c.png)

**Comment by gavrushoves** [comment]
Gulpfile  авторский, без изменений

**Comment by ghost** [comment]
Лучше писать не npm i, а npm install

**Comment by gavrushoves** [comment]
это не помогло

**Comment by ghost** [comment]
Смотрю ошибку пишет по кэшу npm.
Попробуйте npm cache clean --force

**Comment by ghost** [comment]
А, там лог файл просто

**Comment by ghost** [comment]
Node.js свежий?

**Comment by ghost** [comment]
Можно попробовать удалить npm глобально и заново установить

**Comment by gavrushoves** [comment]
ок, спасибо, буду мучать
